src/components/CoreToolkit.tsx

```diff
@@ -0,0 +1,95 @@
+// "use client";
+
+// import classNames from "classnames";
+// import React, { useEffect, useState } from "react";
+
+// function CoreToolkit() {
+//   const [data, setData] = useState([]);
+//   const [hoveredIndex, setHoveredIndex] = useState(null);
+
+//   const getData = () => {
+//     fetch("./output.json", {
+//       headers: {
+//         "Content-Type": "application/json",
+//         Accept: "application/json",
+//       },
+//     })
+//       .then(function (response) {
+//         console.log(response);
+//         return response.json();
+//       })
+//       .then(function (data) {
+//         setData(data);
+//       });
+
+//     console.log("dataaaa", data);
+//   };
+
+//   useEffect(() => {
+//     getData();
+//   }, []);
+
+//   const getAllThemes = data
+//     .map((item, index) => item.PARSED_MANUAL_TAGS.CORE_TOOLKIT)
+//     .flat();
+//   const uniqueThemes = Array.from(new Set(getAllThemes.filter((v) => v)));
+//   console.log("uniqueThemes", uniqueThemes);
+
+//   const coDesignLevelColors = {
+//     1: "bg-red-500",
+//     2: "bg-blue-500",
+//     3: "bg-green-500",
+//   };
+
+//   const handleFilter = (level) => {
+//     setFilterLevel(level);
+//   };
+
+//   return (
+//     <section>
+
+//       <div className="grid grid-cols-2 gap-5 p-20 w-full">
+//         {uniqueThemes.map((item, index) => {
+//           return (
+//               <div
+//               key={index}
+//                 className={classNames({
+//                   "bg-gray-100- border border-dashed p-5 flex flex-col gap-3 bg-gray-100 ": true,
+//     "col-span-3 bg-red-500" : index === 5,
+

```

Comment:
Can we delete this component ? fi we no longer need it

Reply:
Yeap! all unused files are deleted